src/pages/contacts-page/contacts-page.js

```diff
@@ -29,8 +30,10 @@ const pathToAddContactPage = config.routes.pathToAddContact;
 const tableTitles = config.tableHeadRowTitles.contacts;
 
 const ContactsPage = () => {
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed